changelogs/fragments/68288-galaxy-requirements-install-only.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - ansible-galaxy - allow role to define dependency requirements that will be only installed by defining them in ``meta/requirements.ya?ml`` (https://github.com/ansible/proposals/issues/57)
```

Comment:
Typo in `ya?ml` ?

Reply:
Nah it's "regex" like. But I can update to `yml` `yaml` if it's clearer. Both are supported so it made sense to me.

Reply:
my nickel would be to pick one (yml preferrably as that's what the doc uses).